### PR TITLE
Bump rubygem-foreman_rh_cloud to 5.0.30-1

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec [hunk 1]
```diff
@@ -50,7 +50,8 @@ BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 7.8.0
 BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) >= 2.5.0
 BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) < 3.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.14.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 8.17.0
```
Comment (evgeni) [inline packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:54]: in `develop`, we only have `BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.16.0`, without the `<` part -- is the difference intentional?

Comment (ShimShtein) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:54]: It wasn't intentional, it was added automatically.

Comment (Odilhao) [reply packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec:54]: Should we merge with `<` or do a rebase and remove line 54?

**Comment by Odilhao** [comment]
Closing to merge #7529

**Pull request closed without merging**